bin/varnishd/cache/cache_backend.c

```diff
@@ -268,13 +269,15 @@ vbe_dir_gethdrs(VRT_CTX, VCL_BACKEND d)
 	struct pfd *pfd;
 	struct busyobj *bo;
 	struct worker *wrk;
+	enum sess_close sc;
```

Comment:
Doesn't build for me with either GCC or clang:

```
cache/cache_backend.c: In function ‘vbe_dir_gethdrs’
cache/cache_backend.c:272:25: error: storage size of ‘sc’ isn’t known
  272 |         enum sess_close sc;
      |                         ^~
cache/cache_backend.c:272:25: error: unused variable ‘sc’ [-Werror=unused-variable]
```

```
cache/cache_backend.c:272:18: error: variable has incomplete type 'enum sess_close'
        enum sess_close sc;
                        ^
cache/cache_backend.c:272:7: note: forward declaration of 'enum sess_close'
        enum sess_close sc;
             ^
```

Reply:
I just rebased against master to properly solve this conflict, please try again!